README.md

```diff
@@ -22,92 +23,97 @@ documentation, original graphics and other material are released under the
 Licenses of all packages used by Spine Toolbox are listed in the Spine Toolbox User 
 Guide.
 
+## Installation
 
-## Official releases
+Three options for installing Spine toolbox exist: as an application within
+an existing Python installation, using stand-alone installation package and using 
+the source files directly (for developers).
 
-Release versions of Spine Toolbox can be found 
-[here](https://drive.google.com/drive/folders/1t-AIIwRMl3HiYgka4ex5bCccI2gpbspK)
-(only available for 64-bit Windows for now). Download the latest version, install and
-run `spinetoolbox.exe`.
+### Standard Python installation
 
-## Development version
+Spine Toolbox is available on [The Python Package Index (PyPI)](https://pypi.org/project/spinetoolbox/).
+This option is the recommended way to get the most recent release version but a
+working Python installation is also required. 
 
-The development version of Spine Toolbox is in the master branch on this repository, and it has all the latest features and bug-fixes.
-Please note that this version has passed automated testing, but has not been completely manually tested.
+---
+**NOTE:** If you already have Spine toolbox installed globally using pip, please first 
+uninstall using `pip uninstall spinetoolbox`.
 
```

Comment:
Don't they need to activate the environment before running `pip uninstall spinetoolbox`?

In my opinion, all instructions need to be hyper-detailed. We should always assume that users don't know the first thing about python.